tests/test_main.py

```
@@ -1,11 +1,326 @@
-import main
-from datetime import date
-
-class TestMainPipeline(unittest.TestCase):
-    def test_main_dry_run(self):
-        result = main.main(target_date_str=date.today().strftime('%Y-%m-%d'), dry_run=True, force_download=False)
-        self.assertTrue(result)
-
-if __name__ == "__main__":
-    unittest.main()
+import unittest
+from unittest.mock import patch, MagicMock, mock_open, call
```

Ruff (0.11.9)

`unittest.mock.MagicMock` imported but unused

Remove unused import

(F401)